app/components/page-components/Chatbot/ChatbotModules.vue.ts

```diff
@@ -20,6 +20,11 @@ export default class ChatbotModules extends ChatbotBase {
 
   get modules() {
     const backgroundUrlSuffix = this.nightMode ? 'night' : 'day';
+    const comingSoonText = $t(`
+      Streamlabs is diligently working on the next release of Chatbot.
+       Stay tuned.
+       We have more features on the way.
+    `);
```

Comment:
i don't think you can use template literals with the `$t` function, should use single quote strings instead

Reply:
oh interesting. Thanks @gettinToasty